helm/opendistro-es/templates/elasticsearch/es-config-secret.yaml

```diff
@@ -28,4 +28,22 @@ data:
   {{- if .Values.elasticsearch.log4jConfig }}
   log4j2.properties: {{ .Values.elasticsearch.log4jConfig | b64enc | quote }}
   {{- end }}
-  logging.yml: {{ toYaml .Values.elasticsearch.loggingConfig | b64enc | quote }}
+  logging.yml:  {{ toYaml .Values.elasticsearch.loggingConfig | b64enc | quote }}
+
+---
+{{- if .Values.elasticsearch.securityConfig.enabled }}
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ template "opendistro-es.fullname" . }}-es-config-secrets
+  namespace: {{ .Release.Namespace }}
+  labels:
+{{ include "opendistro-es.labels.standard" . | indent 4 }}
+type: Opaque
+data:
+{{- if .Values.elasticsearch.securityConfig.config }}
+{{- range $key, $val := .Values.elasticsearch.securityConfig.config }}
+  {{ $key }}: {{ $val | b64enc | quote }}
+{{- end }}
+{{- end }}
+{{- end}}
```

Comment:
What do you say about moving the security plugin configuration to its own file `es-security-config.yaml`?

Reply:
I think that is a good idea. My first attempt was with a separate yaml file. I will change and push it